### ISSUE TITLE
Fix the Warnings in Build

### ISSUE THREAD

**Issue by rakshit-kumar** [vzhou842/profanity-check#10]
The library is generating lot of warnings (text below).

/Users/sudeepchauhan/Documents/GitHub/announce/venv/lib/python3.7/site-packages/sklearn/utils/deprecation.py:144: FutureWarning: The sklearn.svm.classes module is deprecated in version 0.22 and will be removed in version 0.24. The corresponding classes / functions should instead be imported from sklearn.svm. Anything that cannot be imported from sklearn.svm is now part of the private API.

warnings.warn(message, FutureWarning)

/Users/sudeepchauhan/Documents/GitHub/announce/venv/lib/python3.7/site-packages/sklearn/base.py:318: UserWarning: Trying to unpickle estimator LinearSVC from version 0.20.2 when using version 0.22. This might lead to breaking code or invalid results. Use at your own risk.

UserWarning)

/Users/sudeepchauhan/Documents/GitHub/announce/venv/lib/python3.7/site-packages/sklearn/utils/deprecation.py:144: FutureWarning: The sklearn.preprocessing.label module is deprecated in version 0.22 and will be removed in version 0.24. The corresponding classes / functions should instead be imported from sklearn.preprocessing. Anything that cannot be imported from sklearn.preprocessing is now part of the private API.

warnings.warn(message, FutureWarning)

/Users/sudeepchauhan/Documents/GitHub/announce/venv/lib/python3.7/site-packages/sklearn/base.py:318: UserWarning: Trying to unpickle estimator LabelEncoder from version 0.20.2 when using version 0.22. This might lead to breaking code or invalid results. Use at your own risk.

UserWarning)

/Users/sudeepchauhan/Documents/GitHub/announce/venv/lib/python3.7/site-packages/sklearn/base.py:318: UserWarning: Trying to unpickle estimator _SigmoidCalibration from version 0.20.2 when using version 0.22. This might lead to breaking code or invalid results. Use at your own risk.

UserWarning)

/Users/sudeepchauhan/Documents/GitHub/announce/venv/lib/python3.7/site-packages/sklearn/base.py:318: UserWarning: Trying to unpickle estimator CalibratedClassifierCV from version 0.20.2 when using version 0.22. This might lead to breaking code or invalid results. Use at your own risk.

UserWarning)

**Comment by rakshit-kumar** [comment]
@vzhou842 @vchulski

**Comment by carlinmack** [comment]
Yeah the usage demo produces the following output, meaning I can't use this library :(

```bash
/home/carlin/.local/lib/python3.8/site-packages/sklearn/externals/joblib/__init__.py:15: FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. Please import this functionality directly from joblib, which can be installed with: pip install joblib. If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
  warnings.warn(msg, category=FutureWarning)
/home/carlin/.local/lib/python3.8/site-packages/sklearn/base.py:313: UserWarning: Trying to unpickle estimator CountVectorizer from version 0.20.2 when using version 0.22.2.post1. This might lead to breaking code or invalid results. Use at your own risk.
  warnings.warn(
/home/carlin/.local/lib/python3.8/site-packages/sklearn/utils/deprecation.py:144: FutureWarning: The sklearn.svm.classes module is  deprecated in version 0.22 and will be removed in version 0.24. The corresponding classes / functions should instead be imported from sklearn.svm. Anything that cannot be imported from sklearn.svm is now part of the private API.
  warnings.warn(message, FutureWarning)
/home/carlin/.local/lib/python3.8/site-packages/sklearn/base.py:313: UserWarning: Trying to unpickle estimator LinearSVC from version 0.20.2 when using version 0.22.2.post1. This might lead to breaking code or invalid results. Use at your own risk.
  warnings.warn(
/home/carlin/.local/lib/python3.8/site-packages/sklearn/utils/deprecation.py:144: FutureWarning: The sklearn.preprocessing.label module is  deprecated in version 0.22 and will be removed in version 0.24. The corresponding classes / functions should instead be imported from sklearn.preprocessing. Anything that cannot be imported from sklearn.preprocessing is now part of the private API.
  warnings.warn(message, FutureWarning)
/home/carlin/.local/lib/python3.8/site-packages/sklearn/base.py:313: UserWarning: Trying to unpickle estimator LabelEncoder from version 0.20.2 when using version 0.22.2.post1. This might lead to breaking code or invalid results. Use at your own risk.
  warnings.warn(
/home/carlin/.local/lib/python3.8/site-packages/sklearn/base.py:313: UserWarning: Trying to unpickle estimator _SigmoidCalibration from version 0.20.2 when using version 0.22.2.post1. This might lead to breaking code or invalid results. Use at your own risk.
  warnings.warn(
/home/carlin/.local/lib/python3.8/site-packages/sklearn/base.py:313: UserWarning: Trying to unpickle estimator CalibratedClassifierCV from version 0.20.2 when using version 0.22.2.post1. This might lead to breaking code or invalid results. Use at your own risk.
  warnings.warn(
```

**Comment by workaddiction** [comment]
How do I get rid of this? Would like to use this.

**Comment by jgentil** [comment]
Install `scikit-learn==0.20.2` to solve the problem.

The author really should have specified the exact version.

**Comment by workaddiction** [comment]
> Install `scikit-learn==0.20.2` to solve the problem.
> 
> The author really should have specified the exact version.

It's building wheel (setup.py) endlessly when installing that version using pip3.
